robot/basestation/static/js/rover.js

```diff
@@ -135,36 +135,19 @@ $(document).ready(function () {
     }
   })
 
-  $('#ping-odroid').on('click', function (event) {
-    if (millisSince(lastCmdSent) > PING_THROTTLE_TIME) {
-      appendToConsole('pinging odroid')
-      $.ajax('/ping_rover', {
-        success: function (data) {
-          appendToConsole(data.ping_msg)
-          if (!data.ros_msg.includes('Response')) {
-            appendToConsole('No response from ROS ping_acknowledgment service')
-          } else {
-            appendToConsole(data.ros_msg)
-          }
-        },
-        error: function () {
-          console.log('An error occured')
-        }
-      })
-      lastCmdSent = new Date().getTime()
-    }
+  $('#reboot-button').on('click', function (event) {
+    event.preventDefault()
+    sendRoverCommand('reboot')
   })
-  $('#ping-rover-mcu').on('click', function (event) {
+
+  $('#list-all-cmds').on('click', function(event){
```

Comment:
When I hit the 'L' key I get this error:
Uncaught ReferenceError: disableRoverMotorsBtn is not defined at Object.success (rover.js:437)

Reply:
check around line 437:
```case 76: // 'l' --> list all commands
        lightUp('button#list-all-rover-cmds')
        $.ajax({
          url: '/rover_drive',
          type: 'POST',
          data: {
            cmd: 'l'
          },
          success: function (response) {
            appendToConsole('cmd: ' + response.cmd)
            appendToConsole('feedback:\n' + response.feedback)
            if (!response.feedback.includes('limit exceeded')) {
              disableRoverMotorsBtn()
            }
            if (response.error != 'None') {
              appendToConsole('error:\n' + response.error)
            }
            scrollToBottom()
          }
        })
        lastCmdSent = new Date().getTime()
        break
```

I'm not sure why it is that way right now, maybe because before at some point we used to get the response from the listener script running on the odroid. Instead we should replace that block with the `printCommandsList()` function call.

Reply:
fixed it